admin/bin/qserv-admin.py

```diff
@@ -54,82 +54,94 @@
 class _ToolError(RuntimeError):
     pass
 
+
 class _NotImplementedError(_ToolError):
+
     def __init__(self, cmd):
         _ToolError.__init__(self, "Command not implemented: " + str(cmd))
 
+
 class _IllegalCommandError(_ToolError):
+
     def __init__(self, cmd):
         _ToolError.__init__(self, "Unexpected command or option, see HELP for details: " + str(cmd))
 
+
 class _WrongParamError(_ToolError):
+
     def __init__(self, parm):
         _ToolError.__init__(self, "Unrecognized parameter: " + str(parm))
 
+
 class _MissingParamError(_ToolError):
+
     def __init__(self, parm):
         _ToolError.__init__(self, "Missing parameter: " + str(parm))
 
+
 class _MissingConfigError(_ToolError):
+
     def __init__(self, config):
         _ToolError.__init__(self, "Config file not found: " + str(config))
 
+
 class _UnreadableConfigError(_ToolError):
+
     def __init__(self, config):
         _ToolError.__init__(self, "Can't access the config file: " + str(config))
 
 
-####################################################################################
 class CommandParser(object):
+
     """
     Parse commands and calls appropriate function from qservAdmin.
     """
 
     requiredOpts = {
-        "createDb" : [ # Db opts
+        "createDb": [  # Db opts
             "storageClass",
             "partitioning",
             "partitioningStrategy"
-            ],
-        "createDbSphBox" : [
+        ],
+        "createDbSphBox": [
             "nStripes",
             "nSubStripes",
             "overlap"],
```

Comment:
At some places the closing bracket is on a newline and in other places it is not. Should make it consistent.